test/supports.spec.js

```diff
@@ -85,4 +85,44 @@ describe('supports', function () {
       this.skip()
     }
   })
+
+  it('supportsWebRTCDataChannels should return false in node', function () {
+    if (env.isNode) {
+      expect(supports.supportsWebRTCDataChannels).to.be.false()
+    } else {
+      this.skip()
+    }
+  })
+
+  it('supportsWebRTCDataChannels should return true in browser', function () {
+    if (env.isBrowser) {
+      expect(supports.supportsWebRTCDataChannels).to.be.true()
+    } else {
+      this.skip()
+    }
+  })
+
+  it('supportsWebRTCDataChannels should return true in Web Worker', function () {
```

Comment:
```suggestion
  it('supportsWebRTCDataChannels should return false in Web Worker', function () {
```

Reply:
I've changed the name of the `supportsWebRTC` webworker test as that was wrong too.